mappings/net/minecraft/world/entity/EntityTrackingGroup.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_5572 net/minecraft/world/entity/EntityTrackingGroup
```

Comment:
To carry over from: https://github.com/FabricMC/yarn/pull/1905#discussion_r539274537
"Group" is a very vague name to describe how this class is used. It is specifically used to represent a section within the entity manager. Especially given they reference chunk section state, it makes sense to name according to that.

Proposed names following that discussion were `EntityChunkSection` and `EntityTrackingSection`.

Reply:
group isn't really vague. It is a tracking group where a few entities are tracked together because they are in the same section.

Reply:
Yea, it is an accurate description- but it doesn't give any indication as to why the entities are grouped. There is not really any reason to be less specific.

Reply:
Imo I would keep `Entity` and `Tracking` in the name. After that `Group` or `Section` doesn't really matter; I am currently working on something else. Will come back to this a bit later.

Reply:
Has been renamed @Gegy